charts/keycloak-init/values.yaml

```diff
@@ -657,7 +657,25 @@ keycloak:
             - SUBSCRIBE_OIDC_CLIENT_CREATED_GENERAL
             - SUBSCRIBE_OIDC_CLIENT_UPDATED_GENERAL
             - PUBLISH_AUTHENTICATION_ERRORS_GENERAL
-      users: []
+      users:
```

Comment:
@Mohanraj209 We dont want want this read user as part of default installation.
this should be executed on need basis hence can be moved to the same utils.

Reply:
sure sir